libs/designsystem/src/lib/components/calendar/calendar.component.initialization.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { LOCALE_ID } from '@angular/core';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import moment from 'moment';
+import { MockComponent } from 'ng-mocks';
+
+import { CalendarComponent, IconComponent } from '..';
+import { WindowRef } from '../../types/window-ref';
+
+import { CalendarYearNavigatorConfig } from './options/calendar-year-navigator-config';
+
+describe('CalendarComponent', () => {
+  let spectator: SpectatorHost<CalendarComponent>;
+
+  const createHost = createHostFactory({
+    component: CalendarComponent,
+    declarations: [CalendarComponent, MockComponent(IconComponent)],
+    providers: [
+      {
+        provide: LOCALE_ID,
+        // i.e. en-US. The week should start on Monday regardlessly
+        useValue: 'en',
+      },
+      {
+        provide: WindowRef,
+        useValue: window,
+      },
+    ],
+  });
+
+  beforeEach(() => {
+    spectator = createHost('<kirby-calendar></kirby-calendar>', {
+      props: {
+        minDate: new Date(2017, 0, 1),
+        maxDate: new Date(2025, 11, 31),
+      },
+    });
+  });
+
+  it('should create components with `minDate` and `maxDate` set', () => {
+    expect(spectator.component).toBeTruthy();
+    expect(spectator.component.minDate).toEqual(new Date(2017, 0, 1));
+    expect(spectator.component.maxDate).toEqual(new Date(2025, 11, 31));
+  });
+});
```

Comment:
I decided on this solution because spectator does not allow multiple instances of a host during a test. The alternative to this was to make activeMonth public and then test directly on activeMonth, however I felt like that was too big a change. I'm open to better solutions